### DISCUSSION TITLE
Error performing ivabs example

### DISCUSSION THREAD

**Discussion by kimjihun8** [wenbinyugroup/.github#21] [VABS Related Discussions]
I'm doing an opt_cs_box_lam example of ivabs (ivabs-examples-0.11.build20240623.1). I'm using vabs v4.0.

dakota is using version 6.20 and python is using the latest version of anaconda.

I have completed setting up path/python path of ivabs and dakota as shown in the attached picture.
![error](https://github.com/user-attachments/assets/c989b2c4-352c-4302-b7b7-62da6665b80c)

When you perform the interpretation, a message appears as shown in the figure and does not proceed properly from eval.1.
![setting](https://github.com/user-attachments/assets/32584db9-7f86-447a-bf04-049606ddd77f)

Is there a solution?

**Comment by unitasium** [comment]
Does the attached screenshot show the complete error message?

**Reply by kimjihun8** [reply]
![image](https://github.com/user-attachments/assets/d2e090d4-54e9-484b-8329-32bb3ab80c40)
![image](https://github.com/user-attachments/assets/0e8f0490-f1e9-4da3-b28b-a57f3866bd29)
![image](https://github.com/user-attachments/assets/bdee875d-2641-4cbe-b9df-3e4f8998b2a4)

**Reply by kimjihun8** [reply]
Due to compatibility issues with the numpy 2.0 and later versions, we modified the file _med.py to numpy_void_str = np.string_(") ==> numpy_void_str = np.bytes_(") to try with the example of anl_blade_cs_rcas.

The ivabs annalyze main.yml command is executed normally, but the ivabs plot --sgdirks main.yml command is not executed normally.
![image](https://github.com/user-attachments/assets/cebf4118-7327-4856-b07c-cf466dc4c7d8)
